libs/core/src/lib/micro-process-flow/components/micro-process-flow/micro-process-flow.component.scss

```diff
@@ -0,0 +1,214 @@
+@import "../../mixins.scss";
+@import "../../variables.scss";
+
+// TODO: Move all styles to fundamental-styles library
+
+.#{$block} {
+    @include fd-reset();
+    @include fd-flex();
+
+    max-width: $max-container-width;
+    overflow: hidden;
+
+    &--independent-steps {
+        .#{$block}__connector {
+            &:before {
+                border-color: transparent;
+            }
+        }
+    }
+}
+
+.#{$block}__wrapper {
+    @include fd-reset();
+    @include fd-flex();
+
+    //transition: transform .3s ease;
+}
+
+.#{$block}__overflow-container {
+    @include fd-reset();
+    @include fd-flex();
+    width: 100%;
+    overflow: hidden;
+}
+
+.#{$block}__link-previous, .#{$block}__link-next {
+    @include fd-reset();
+    @include fd-flex();
+    justify-content: flex-start;
+    align-items: stretch;
+
+    min-width: 2rem;
+    cursor: pointer;
+
+    span {
+        @include fd-reset();
+        color: var(--sapGroup_TitleTextColor);
+        text-shadow: var(--sapContent_TextShadow);
+        font-size: var(--sapFontSize);
+        @include fd-flex();
+        justify-content: center;
+        align-items: center;
+    }
+
+    &:before, &:after {
+        font-family: SAP-icons;
+        text-align: center;
+        text-decoration: inherit;
+        text-transform: none;
+        text-rendering: optimizeLegibility;
+        @include fd-flex();
+        justify-content: center;
+        align-items: center;
+
+        color: var(--sapButton_TextColor);
+    }
+
+    @include fd-hover {
+        color: var(--sapButton_Hover_TextColor);
+    }
+}
+
+.#{$block}__link-previous {
+
+    &:before {
+        content: "\e067";
+        padding-right: 0.313rem;
```

Comment:
I recommend copying the `fd-set-padding-left` and `fd-set-padding-right` mixins from fund-styles. It will save you the rtl handle and adding a variable for the padding value.